scripts/README.md

```diff
@@ -0,0 +1,255 @@
+# scripts
+
+The TANF app uses several scripts through its lifecycle. 
+These don't all get used by or interacted with by us too often,
+but some are mission critical during deployment and review of 
+the application. When a developer is working on these scripts, 
+they should update this documentation so future developer's 
+understand the role of the scripts
+
+# set-backend-env-vars.sh 
+
+## usage 
+
+```bash
+
+./scripts/set-backend-env-vars.sh CGAPPNAME_BACKEND CG_SPACE
+```
+
+## args
+
+`CGAPPNAME_BACKEND` the sub domain of the app you are trying to set up environment variables for.
+`CG_SPACE` the space the app you are trying to set up for exists in.
+
+## description
+
+Determine the appropriate BASE_URL for the deployed instance based on the
+provided Cloud.gov App Name
+Use Shell Parameter Expansion to replace localhost in the URL
+Dynamically generate a new `DJANGO_SECRET_KEY`
+Dynamically set `DJANGO_CONFIGURATION` based on Cloud.gov Space
+
+# sudo-check.sh
+
+## usage
+
+```
+./scripts/sudo-check.sh
+
+```
+
+## Args
+
+no args
+
+## description
+
+check if the `sudo` command is present, installs it if it isn't.
```

Comment:
Please provide the reader context that these "check" scripts are used by our CircleCI pipeline and that they are ran inside our buildpack containers. Additionally, please use formal grammar and full-length sentences in descriptions.